data/json/items/tool/workshop.json

```diff
@@ -1164,9 +1164,9 @@
     ]
   },
   {
-    "id": "tire_iron",
+    "id": "lug_wrench",
```

Comment:
I understand changing the item's translated name, but should the ID really be changed? It's not user-facing data, and some 3rd party mods might be relying of the ID. Not sure about this, but feel free to correct me

Reply:
I went back and forth on whether to change the name at all; the thing that pushed me over the edge was that I wouldn't be surprised if someone wanted to add **actual** tire irons within the year. If they do, what ID are they going to use? Sure, they can just use `tire_level` or `tire_spoon` or `tire_iron_actual` or something, but if this is still tire_iron when they do, doesn't it seem likely that future coders will conflate the two items, possibly even creating more bugs in the process? I know these are all hypotheticals but imo these things are important to nip in the bud early, before they have a chance to accrue more technical debt.

If the mergers agree with you and not me, though, I will happily concede this point. There's even a third option, which is to add a duplicate `tire_iron` item to `obsolete.json`.

Reply:
That makes sense. Cheers!

Reply:
For sure 😄 thanks for the review!